large-message-serde/src/test/java/com/bakdata/kafka/LargeMessageSerializerTest.java

```diff
@@ -153,6 +167,30 @@ private static void expectNonBackedText(final String expected, final byte[] s3Ba
         assertThat(headers.headers(getHeaderName(isKey))).hasSize(1);
     }
 
+    private static void deleteBucket(final String bucket, final AmazonS3 s3Client) {
```

Comment:
Maybe you could look into extracting common test setup and utilities into a test-fixture, don't know if it's already worth it though

Reply:
I switched to Localstack. Works much better than S3Mock